velox/functions/sparksql/tests/BitwiseTest.cpp

```diff
@@ -46,6 +46,16 @@ class BitwiseTest : public SparkFunctionBaseTest {
     return evaluateOnce<T>("bitwise_or(c0, c1)", a, b);
   }
 
+  template <typename T>
+  std::optional<T> bitwiseXor(std::optional<T> a, std::optional<T> b) {
```

Comment:
nit: maybe it's better to arrange the test methods and unit tests in alphabetical order.

Reply:
updated, thanks.